examples/shell/main.cpp

```diff
@@ -39,10 +40,14 @@ void cmd_otcli_init();
 
 int main(void)
 {
+    // Initialize the default streamer that was linked.
+    assert(streamer_init(streamer_get()) == 0);
```

Comment:
side effects in assert() will break in release builds

Reply:
Right, fixed it already. Thank you!